plume-models/src/users.rs

```diff
@@ -577,6 +577,28 @@ impl User {
             .map_err(Error::from)
     }
 
+    pub fn count_followed(&self, conn: &Connection) -> Result<i64> {
```

Comment:
n.b.: many people on the fediverse are not happy with the "clout" that comes from these numbers

do we offer an option to disable it?

i would also suggest considering "subscribe" instead of "follow".
it fits better with a blog, and has slightly less of a cult meaning

Reply:
These numbers are not currently displayed in the UI, we only use this function to know how many pages listing followers/followed we need. If we display them one day, I totally agree that we should be able to hide them (maybe even hide them by default?), or that they should only be shown for your own profile.

And I agree on "follow"/"subscribe" too, I will change it. :slightly_smiling_face: